### ISSUE TITLE
Trace imports are invisible

### ISSUE THREAD

**Issue by Kixiron** [TimelyDataflow/differential-dataflow#330]
No events are emitted that can be used to piece together where an imported trace came from, `TraceAgent`s manage their data internally and `.import()` uses the `source` operator to create an entirely new source, so there's no `ChannelsEvent` linking the two streams nor any sort of differential event that links them.

**Comment by frankmcsherry** [comment]
Just to make sure that I understand the issue, trace imports are reported (the `TraceShare` logging event), but not enough breadcrumbs are reported to let you link up the the imported arrangement with the location it was imported to?

**Comment by Kixiron** [comment]
Yep, precisely. `TraceShare` just says that an arrangement came from somewhere, but not where it actually comes from

**Comment by frankmcsherry** [comment]
I think it might be the other way around. `TraceAgent` records the operator id it was constructed with, and on each importing event increments that. So, each import does announce where it came from, but only where it came from (not, e.g. where it was imported to). Maybe that is what you meant though.

**Comment by Kixiron** [comment]
Yes, that's my bad
